modules/engine/src/debug/debug-framebuffer.ts

```diff
@@ -0,0 +1,57 @@
+import type {Framebuffer, Texture} from '@luma.gl/core';
+// import {copyTextureToImage} from '../debug/copy-texture-to-image';
+
+/** Only works with 1st device? */
+let canvas: HTMLCanvasElement | null = null;
+let ctx: CanvasRenderingContext2D | null = null;
+// let targetImage: HTMLImageElement | null = null;
+
+/** Debug utility to draw FBO contents onto screen */
+// eslint-disable-next-line
+export function debugFramebuffer(
```

Comment:
A lot of this could be built into luma.gl. 

One thing is that this logging is very slow, and doing it for all framebuffers with a single flag may not always be the best approach. We may want to add a `.debug` field to the FBO to enable display for just that framebuffer, and potentially this could be an object with options. 

If we truly need it to be configurable we can consider that.

Also for WebGPU we probaly want to put this on the RenderPass on the FBO, as the end of the renderpass drives the rendering.

Reply:
Agreed on all points - I haven't particularly needed the ability to debug multiple framebuffers at once, so far, just the ability to choose which is debugged.